src/ServiceManagement/Common/Commands.Common.Test/Common/ProfileClientTests.cs

```diff
@@ -1503,7 +1503,7 @@ public void ImportPublishSettingsAddsSecondCertificate()
         private void SetMocks(List<RDFESubscription> rdfeSubscriptions,
             List<CSMSubscription> csmSubscriptions,
             List<TenantIdDescription> tenants = null,
-            Func<IAzureAccount, IAzureEnvironment, string, IAccessToken> tokenProvider = null)
+            Func<IAzureAccount, IAzureEnvironment, string, IRenewableToken> tokenProvider = null)
         {
             ClientMocks clientMocks = new ClientMocks(new Guid(defaultSubscription));
```

Comment:
Please also add in some validation of the new property in our mocked tests of the authentication factory.

Reply:
Done